digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -523,6 +523,20 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
 
   def preSplitSuravageLinkInTX(linkId: Long, username: String,
                                splitOptions: SplitOptions): (Option[Seq[ProjectLink]], Option[String], Option[(Point, Vector3d)]) = {
+    //Discontinuity of splits should be the given only for the part with the biggest M Address Value
```

Comment:
I think this would better fit ProjectLinkSplitter.scala where all the other adjustment logic resides too. For example     
def switchDigitization(splits: (ProjectLink, ProjectLink, ProjectLink)) = ...

Reply:
I agree, switching.